config/fcos/v1_6_exp/validate.go

```diff
@@ -51,3 +51,23 @@ func (user GrubUser) Validate(c path.ContextPath) (r report.Report) {
 	}
 	return
 }
+
+func isMountpointForbidden(path string) bool {
+	// Check if the path is a restricted directory
+	restrictedDirs := []string{"/boot", "/sys", "/usr", "/var", "/", "/sysroot/mnt"}
+	for _, dir := range restrictedDirs {
+		if path == dir {
+			return true
+		}
+	}
+	return false
+}
+
+func (c Config) ValidateMountPoints() (r report.Report) {
+	for m, mountPoint := range c.Storage.Filesystems {
+		if isMountpointForbidden(*mountPoint.Path) {
+			r.AddOnError(path.New("forbidden_mountpoint", m), common.ErrMountPointForbidden)
```

Comment:
I think the correct path is more like `path.new("filesystems",i,"path")`

Reply:
I like to think of the path as a map back to the problem area of the config for the end user to debug.